src/components/settings/NonTandem.css

```diff
@@ -70,3 +71,39 @@
 .bolusSettingsHeader {
   composes: bolusHeaderBackground settingsHeader from './settings.css';
 }
+
+@media print {
+
+  .settingsContainer {
+    display: block;
+  }
+
+  .categoryContainer {
+    margin-bottom: 10px;
+    margin-right: 10px;
+  }
+
+  .basalSettingsContainer {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-around;
+  }
+
+  .bolusSettingsContainer {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-around;
+  }
```

Comment:
I think this and the `.basalSettingsContainer` above could both use `page-break-inside: avoid`s.